.github/workflows/ci.yml

```diff
@@ -122,7 +122,7 @@ jobs:
           # Enable auto-creation of topics
           KAFKA_CFG_AUTO_CREATE_TOPICS_ENABLE: 'true'
         ports:
-          - '9092:9092'
+          - '19092:19092'
```

Comment:
question: why can't it remain on 19092? The controller is still on 9093

Reply:
Irrelevant now, as I have simplified the config.